README.md

```diff
@@ -1,3 +1,71 @@
-﻿# KMD Logic Email Api Client
+﻿# KMD Logic Email Api Client ![Email icon](Email.png)
 
 A dotnet client library for using Email module in KMD Logic platform through API.
+## The purpose of the Email API
+
+To allow products using the Logic platform to **send email using custom email domain address**  
```

Comment:
Don't think there is a need to mention custom domain address. `To allow products using Logic platform to send emails` should be enough.

Reply:
Done